packages/amplify-appsync-simulator/package.json

```diff
@@ -90,4 +90,4 @@
       "node"
     ]
   }
-}
+}
```

Comment:
Do you have a VSCode setting that is removing newlines at the end of JSON files? Ideally all text files should end in a newline